README.md

```diff
@@ -114,45 +114,15 @@ Version: 3
 
 </details>
 
-### Google Play Store **(Deprecated)**
+### Google Play Store **(Experimental branch)**
 
-**Termux and its plugins are no longer updated on [Google Play Store](https://play.google.com/store/apps/details?id=com.termux) due to [android 10 issues](https://github.com/termux/termux-packages/wiki/Termux-and-Android-10) and have been deprecated.** The last version released for Android `>= 7` was `v0.101`. **It is highly recommended to not install Termux apps from Play Store any more.**
+Most users should use the F-Droid or GitHub installation source as described above.
```

Comment:
Since I shortened info below I suggest here instead:

`Most users should use the F-Droid or GitHub installation source as described above, to get a stable experience and access to all packages.`